ert_shared/ensemble_evaluator/context_manager.py

```diff
@@ -44,6 +45,9 @@ def _attach(run_context, run_path_list, forward_model):
     logger.debug("ee ws started")
 
     event_logs = [Path(path.runpath) / "event_log" for path in run_path_list]
+    for event_log in event_logs:
```

Comment:
Isn't this file being cleared anyway?

Reply:
Yes, but the problem is that the NSF adaptor reads from the file before the job_dispatch clears it 😅

Reply:
True. 😂 Well, if these two components (job_dispatch and nfs_adaptor) are to become _more_ concurrent, this strategy will not work. Maybe we should just focus on the `source` of the events, and parse the `ee` bit. If there's a mismatch, then drop the event?

Reply:
(Some time in the future)